loadtest.py

```diff
@@ -69,19 +69,27 @@ def _run():
 async def test(session):
     storage = session.storage
 
+    # Respect the server limits.
+    _, config = await storage.get("/info/configuration")
+    print("Config: {}".format(json.dumps(config, indent=3)))
```

Comment:
this is per each individual test run so this'll be too chatty, let's comment it out